### PR TITLE
feat(cli): add migration from 2.0.0-beta to 2.0.0-rc

### PR DESCRIPTION
**Pull request by lucasfernog**
Includes migration for #10390

### CODE REVIEW

**Comment by lucasfernog** [comment]
it's this one @FabianLars https://rust-lang.github.io/rust-clippy/master/index.html#/doc_lazy_continuation

File: tooling/cli/src/migrate/migrations/v2_rc.rs [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+fn migrate_permissions(tauri_dir: &Path) -> Result<()> {
+  let core_plugins = [
+    "app",
+    "event",
+    "image",
+    "menu",
+    "resources",
+    "tray",
+    "webview",
+    "window",
```
Comment (tillmann-crabnebula) [inline tooling/cli/src/migrate/migrations/v2_rc.rs:99]: I think that is missing the core pseudo `path` plugin

Comment (lucasfernog) [reply tooling/cli/src/migrate/migrations/v2_rc.rs:99]: of course, i got it from the other PR 😂

**Review by tweidinger — APPROVED** [review]
Haven't tested and only reviewed the code but LGTM 🙃

**Pull request merged**